src/main/java/com/gdsc/studyjamapi/controller/UserController.java

```diff
@@ -25,6 +26,13 @@ public ResponseEntity<UserResponse> createUser(
     return ResponseEntity.created(locationOfNewUser).build();
   }
 
+  public ResponseEntity<UserResponse> editUser(
+          EditUserRequest request, UriComponentsBuilder ucb) {
+    UserResponse editUser = userService.editUser(request);
+    URI locationOfNewUser = ucb.path("/users/{id}").buildAndExpand(editUser.getId()).toUri();
+    return ResponseEntity.created(locationOfNewUser).build();
```

Comment:
You should review the convention on PUT APIs. It doesn't return the 201 Created code as you did here.